### PR TITLE
jf/bisector_example

### PR DESCRIPTION
**Pull request by jf---**

### CODE REVIEW

File: examples/core_geometry_bisector.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import os
```
Comment (tpaviot) [inline examples/core_geometry_bisector.py]: Please insert a header

File: examples/core_geometry_bisector.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import sys
+from OCC.Bisector import Bisector_BisecCC
+from OCC.Display.SimpleGui import init_display
+
+# start gui
```
Comment (tpaviot) [inline examples/core_geometry_bisector.py]: remove this line and move up the content

File: examples/core_geometry_bisector.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+from OCC.GccAna import GccAna_Lin2dBisec, GccAna_CircLin2dBisec, GccAna_Pnt2dBisec
+from OCC.gp import gp_Lin2d, gp_Pnt2d, gp_Dir2d, gp_Circ2d, gp_Ax22d, gp_Pnt
+
+display, start_display, add_menu, add_function_to_menu = init_display()
+
+sys.path.append(os.path.dirname(__file__))
+
+from core_geometry_utils import make_vertex, make_edge2d
+
+
```
Comment (tpaviot) [inline examples/core_geometry_bisector.py]: remove one blank line (2 are enough)

File: examples/core_geometry_bisector.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    display.FitAll()
+
+
+def bisect_linecircle(event=None):
+    display.EraseAll()
+    ci1 = gp_Circ2d(gp_Ax22d(), 10000)
+    li1 = gp_Lin2d(gp_Pnt2d(2000000, 20), gp_Dir2d(0, 1))
+    bi = GccAna_CircLin2dBisec(ci1, li1)
+    assert bi.IsDone()
+    aaa = bi.ThisSolution(1).GetObject()
```
Comment (tpaviot) [inline examples/core_geometry_bisector.py]: please use explicit variable name ('aaa' is not explicit)

File: examples/core_geometry_bisector.py [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    pb = aaa.Parabola()
+    display.DisplayShape([make_edge2d(ci1), make_edge2d(li1)])  # make_edge2d(li1)
```
Comment (tpaviot) [inline examples/core_geometry_bisector.py]: remove comment at the end of the line

File: examples/core_geometry_bisector.py [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    display.DisplayColoredShape(make_edge2d(pb), 'BLUE')
+    display.FitAll()
+
+
+def bisect_pnt(event=None):
+    display.EraseAll()
+    p1 = gp_Pnt2d(1, 0.5)
+    p2 = gp_Pnt2d(0, 1e5)
+    bi = GccAna_Pnt2dBisec(p1, p2)
+    aaa = bi.ThisSolution()
```
Comment (tpaviot) [inline examples/core_geometry_bisector.py]: please use an explicit variable name ('aaa' is not explicit)

File: examples/core_geometry_bisector.py [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+    display.FitAll()
+
+
+if __name__ == '__main__':
+    add_menu('bisector')
+    add_function_to_menu('bisector', bisect_lineline)
+    add_function_to_menu('bisector', bisect_linecircle)
+    add_function_to_menu('bisector', bisect_pnt)
+    add_function_to_menu('bisector', bisect_crvcrv)
+
```
Comment (tpaviot) [inline examples/core_geometry_bisector.py]: remove blank line to be constent with other examples

**Comment by tpaviot** [comment]
@jf--- Tested, it works. Just a few aesthetic changes requested.

**Comment by jf---** [comment]
many thanks for the review @tpaviot, that's much appreciated!

**Pull request merged**